students/10th/soomiHwang/repl/user/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views  import SignUp, SignIn
+
+urlpatterns = [
+    path('/SignUp', SignUp.as_view()),
+    path('/SignIn', SignIn.as_view())
```

Comment:
url은 시작을 대문자로 하지는 않습니다. 따라서 다음과같이 수정 부탁 드립니다.
클래스 명을 대문자로 시작하고 카멜케이스로 합니다.
```suggestion
    path('/sign-up', SignUp.as_view()),
    path('/sign-in', SignIn.as_view())
```

Reply:
넵~!